ansible/group_vars/osdsdock.yml

```diff
@@ -65,6 +65,10 @@ ceph_description: This is a ceph backend service
 ceph_driver_name: ceph
 ceph_config_path: "{{ opensds_driver_config_dir }}/ceph.yaml"
 
+ceph_ansible_branch: stable-3.0
```

Comment:
Query: Why don't we use stable-3.1? It will support both luminous and mimic. 
Also, with major version released (i.e. 4.0) and stable. It's wiser to move ahead to atleast 3.1, IMHO

Reply:
As for me, it's up the community, and I took 'stable-3.0' as default so that this PR does not change the overall behavior greatly. But, anyway I also think 'stable-3.1' is a good choice because it does not change ansible version and supports both luminous and mimic. If other forks are OK, let's switch to 'stable-3.1'.

Reply:
Yes, We need to support the ceph version in accordance to ansible version

Reply:
Yes, 3.1 won’t affect ansible version. We can put it on slack or mailing list. If someone has different opinion then we will think